mxd/backend-service/src/main/java/org/eclipse/mxd/service/Impl/TransferServiceImpl.java

```diff
@@ -0,0 +1,154 @@
+                ErrorModel errorModel = new ErrorModel(500, "Internal Server Error", e.getMessage());
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            } catch (Exception ex) {
+                logger.info(ex.getMessage());
+                return Response.status(Response.Status.BAD_REQUEST)
+                        .entity("Internal Server Error")
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        }
+
+    }
+
+    @Override
+    public Response getTransfer(String id) {
+        try {
+            TransfersModel transfersModel = this.transferRepository.getTransferById(id);
+            if (transfersModel != null && transfersModel.getId() != null) {
+                JsonNode jsonNode = new ObjectMapper().readTree(transfersModel.getAsset());
+                TransferModelsResponse transferModelsResponse = new TransferModelsResponse();
+                transferModelsResponse.setAsset(jsonNode);
+                return Response.ok(new ObjectMapper().writeValueAsString(transferModelsResponse.getAsset())).build();
+            } else {
+                ErrorModel errorModel = new ErrorModel(404, "Resource Not Found", "Requested Resource Not Found");
+                return Response.status(Response.Status.NOT_FOUND)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        } catch (Exception e) {
+            try {
+                logger.info(e.getMessage());
+                ErrorModel errorModel = new ErrorModel(500, "Internal Server Error", e.getMessage());
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            } catch (Exception ex) {
+                logger.info(ex.getMessage());
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity("Internal Server Error")
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        }
+    }
+
+    @Override
+    public Response getTransferContents(String id) {
+        try {
+            TransfersModel transfersModel = this.transferRepository.getTransferById(id);
+            if (transfersModel != null && transfersModel.getContents() != null) {
+                JsonNode jsonNode = new ObjectMapper().readTree(transfersModel.getContents());
+                TransferModelsResponse transferModelsResponse = new TransferModelsResponse();
```

Comment:
No need of `TransferModelsResponse`, use jsonNode directly.

Reply:
I have used  jsonNode directly as suggested.